getting_started/workflow/assets/importing_audio_samples.rst

```diff
@@ -7,42 +7,42 @@ Why import?
 -----------
 
 Raw audio data in general is large and undesired. Godot provides two main
-options to import your audio data: WAV and OGG Vorbis.
+options to import your audio data: WAV and Ogg Vorbis.
 
 Each has different advantages.
 
-* Wav files use raw data or light compression, make few demands on the CPU to play back (hundreds of simultaneous voices in this format are fine), but take up significant space.
+* WAV files use raw data or light compression, make few demands on the CPU to play back (hundreds of simultaneous voices in this format are fine), but take up significant space.
 * Ogg Vorbis files use a stronger compression that results in much smaller file size, but require significantly more processing power to play back.
 
 
 .. image:: img/audio_stream_import.png
 
 Here is a comparative chart.
 
-+-------------------------------+---------------------+
-| Format                        | 1 Second of Audio   |
-+===============================+=====================+
-| WAV 24 bits, 96 kHz, Stereo   | 576kb               |
-+-------------------------------+---------------------+
-| WAV 16 bits, 44 kHz, Mono     | 88kb                |
-+-------------------------------+---------------------+
-| WAV 16 bits, IMA-ADPCM, Mono  | 22kb                |
-+-------------------------------+---------------------+
-| OGG 128kbps, Stereo           | 16kb                |
-+-------------------------------+---------------------+
-| OGG Vorbis 96kbps, Stereo     | 12kb                |
-+-------------------------------+---------------------+
-
-Consider using WAV for short and repetitive sound effects, and OGG
++-----------------------------+-------------------+
+| Format                      | 1 second of audio |
++=============================+===================+
+| WAV 24-bit, 96 kHz, stereo  | 576 KB            |
++-----------------------------+-------------------+
+| WAV 16-bit, 44 kHz, mono    | 88 KB             |
++-----------------------------+-------------------+
+| WAV 16-bit, IMA-ADPCM, mono | 22 KB             |
++-----------------------------+-------------------+
+| Ogg Vorbis 128 Kb/s, stereo | 16 KB             |
++-----------------------------+-------------------+
+| Ogg Vorbis 96 Kb/s, stereo  | 12 KB             |
++-----------------------------+-------------------+
+
+Consider using WAV for short and repetitive sound effects, and Ogg Vorbis
 for music, speech, and long sound effects.
 
 Best practices
 --------------
 
-Godot 3+ has an amazing bus system with built-in effects. This saves
-SFX artists the need to add reverb to the sound effects, reducing their
-size greatly and ensuring correct trimming. Say no to SFX with baked
-reverb!
+Godot has an :ref:`amazing bus system <doc_audio_buses>` with built-in effects.
```

Comment:
How about "extensive bus system"? "amazing bus system" feels like someting you'd find in an advertisment in my opinion. Sounds fancy but doesn't tell you anything.

Reply:
Agreed :)